### PR TITLE
Add `GB Cup 2024 IRL` wiki articles

### PR DESCRIPTION
**Pull request by yobrevelc**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [X] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

`SKIP_OUTDATED_CHECK`

### CODE REVIEW

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+### Regular Stage
+
+Due to the large number of players and matches, the match results will be displayed in the **[information spreadsheet](https://docs.qq.com/sheet/DTWpxUkN6SGtrS29m?tab=j9vlga)**.
+
+## Qualifier instructions
+
+1. There are 8 maps in the qualifier mappool.
+2. The ranking of qualifier is determined by sum of all the single maps ranking.
+3. The top 16 players in the qualifier will enter the main competition. They will be divided into 4 tiers according to the player ranking and randomly assigned into 4 quarter zones, while every quarter will forcely include 1 player from each tier. The first round of the game is fixed as tier 1 vs tier 4, and tier 2 vs tier 3.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:145]:
```suggestion
3. The top 16 players in the qualifier will enter the main competition. They will be divided into 4 tiers according to the player ranking and randomly assigned into 4 quarter zones, where every quarter will forcibly include 1 player from each tier. The first round of the game is fixed as tier 1 vs. tier 4, and tier 2 vs. tier 3.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+4. After the qualifier round, if any qualified player withdraws, his ranking will be replaced by the next player, and so on.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:146]:
```suggestion
4. After the qualifier round, if any qualified player withdraws, their ranking will be replaced by the next player, and so on.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+## Tournament rules
+
+1. There is no warm-up phase for the match.
+2. The main competition is a single-elimination knockout competition with a total of 16 players. The round of 16 is a best-of-seven game, the quarterfinals and semifinals are a best-of-nine game, and the final is a best-of-eleven game.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:151]:
```suggestion
2. The main competition is a single-elimination knockout competition with a total of 16 players. The Round of 16 is a best-of-7 game, the Quarterfinals and Semifinals are a best-of-9 game, and the Final is a best-of-11 game.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+3. There are three maps for each slot in the mappool. When selecting a map, you can only select the position. The specific map to be used will be determined by random roll on the spot.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:152]:
```suggestion
3. There are three maps for each slot in the mappool. When selecting a map, you can only select the position. The specific map to be used will be determined by random `!roll` on the spot.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+4. During the protect phase, both players choose a slot and assign it to a specific map. The map of this slot will not be randomized in subsequent selections, if this slot got piced the map will directly be selected. Also this slot cannot be banned.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:153]:
```suggestion
4. During the protect phase, both players choose a slot and assign it to a specific map. The map of this slot will not be randomised in subsequent selections. If this slot gets picked, the map will directly be selected. Additionally, this slot cannot be banned.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+5. During the ban phase, both players will ban all the maps of one slot.
+6. When the two sides are tied and the last point is decided, the last map will use Tie-breaker. At this time, both sides will ban one map at the same time through the referee. If different maps are banned, the remaining one will be the map; if the same map is banned, one map will be randomly selected from the remaining two maps.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:155]:
```suggestion
6. When the two sides are tied and the last point is decided, the last map will be a tiebreaker. At this time, both sides will ban one map at the same time through the referee. If different maps are banned, the remaining one will be the map played; if the same map is banned, one map will be randomly selected from the remaining two maps.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+7. Each side will have 2 minutes to choose a map. If the timeout is reached, the referee will randomly select an unselected map from the mappool. After the map is selected, both players have 2 minutes to get ready.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:156]: written out elsewhere in this article

```suggestion
7. Each side will have two minutes to choose a map. If the timeout is reached, the referee will randomly select an unselected map from the mappool. After the map is selected, both players have two minutes to get ready.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
+---
+tags:
+  - GBC
+  - GBC2024
+  - GBC 2024
+  - GBC 2024 IRL
+  - GBC2024IRL
+---
+
+# # GB Cup 2024 In Real Life
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:10]:
```suggestion
# GB Cup 2024 In Real Life
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+5. During the ban phase, both players will ban all the maps of one slot.
+6. When the two sides are tied and the last point is decided, the last map will use Tie-breaker. At this time, both sides will ban one map at the same time through the referee. If different maps are banned, the remaining one will be the map; if the same map is banned, one map will be randomly selected from the remaining two maps.
+7. Each side will have 2 minutes to choose a map. If the timeout is reached, the referee will randomly select an unselected map from the mappool. After the map is selected, both players have 2 minutes to get ready.
+8. Possible mod choices for the FreeMod bracket are NoFail, Hidden, Fade In, Flashlight, and Mirror.
+9. There are no restrictions on whether both players can choose or not choose a certain map, except that the selected map must be in the game map pool, and you cannot choose a map that has been selected in the game, and you cannot choose Tiebreaker.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:158]:
```suggestion
9. There are no restrictions on whether both players can choose or not choose a certain map, except that the selected map must be in the game mappool, and you cannot choose a map that has been selected in the game, and you cannot choose the tiebreaker.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+10. In the case of a tie, the beatmap will be rematched, but not more than twice.
+11. Players are allowed to rematch if any player disconnects early on during a beatmap. Send a message as soon as poosible in the match room chat to notify the referee to restart the match.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:160]:
```suggestion
11. Players are allowed to rematch if any player disconnects early on during a beatmap. Send a message as soon as possible in the match room chat to notify the referee to restart the match.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+12. If necessary, the referee has the right to decide the implementation of the rules at his own discretion and has the final say on events that occur during the game.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:161]:
```suggestion
12. If necessary, the referee has the right to decide the implementation of the rules at their own discretion and has the final say on events that occur during the game.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
the GBC index page needs an update

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 12]
```diff
@@ -0,0 +1,161 @@
...
+  - GBC 2024
+  - GBC 2024 IRL
+  - GBC2024IRL
+---
+
+# # GB Cup 2024 In Real Life
+
+![GB杯2024线下赛图片](img/gbc2024irl-banner.png)
+
+**GBC 2024 IRL** (***GB Cup 2024 In Real Life***) is hosted by [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) and organized by GB. It is an osu!mania 4k event that was held offline at the Kululu Music Game Carnival.
```
[the remark this answers is not in the document]
Comment (yobrevelc) [reply wiki/Tournaments/GBC/GBC_2024_IRL/en.md:14]: according to my knowledge there is no link of Kululu Music Game Carnival, only some videos I guess

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+  - GBC 2024
+  - GBC 2024 IRL
+  - GBC2024IRL
+---
+
+# GB Cup 2024 In Real Life
+
+![GB杯2024线下赛图片](img/gbc2024irl-banner.png)
+
+**GB Cup 2024 In Real Life** (***GBC 2024 IRL***) is a LAN osu!mania 4-key solo tournament hosted by [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) and organized by GB. It was the sixth instalment of the GB Cup series and the first instalment of the GB Cup IRL. This LAN tournament was held offline at the Kululu Music Game Carnival in Shanghai, China.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:14]:
```suggestion
**GB Cup 2024 In Real Life** (***GBC 2024 IRL***) is a osu!mania 4-key solo LAN tournament hosted by [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) and organised by Team GB. It was the sixth instalment of the GB Cup series and the first instalment of the GB Cup IRL. This LAN tournament was held offline at the Kululu Music Game Carnival in Shanghai, China.
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 14]
```diff
@@ -0,0 +1,166 @@
...
+| Placing | Prizes |
+| :-: | :-- |
+| ![Golden Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, championship trophy, GBC pillow, GBC medal |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | KanonBot profile badge, 2nd place trophy, GBC pillow, GBC medal |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | KanonBot profile badge, 3rd place trophy, GBC pillow, GBC medal |
+| *Others* | GBC pillow, GBC medal |
+
+![Trophy Image](img/gbc2024irl-cups.jpg "GBC 2024 IRL trophies")
+
+![Badge Image](img/gbc2024irl-winner.png)
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:35]:
```suggestion
![Trophy image](img/gbc2024irl-cups.jpg "GBC 2024 IRL trophies")

![Badge image](img/gbc2024irl-winner.png)
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 15]
```diff
@@ -0,0 +1,166 @@
...
+  - GBC
+  - GBC2024
+  - GBC 2024
+  - GBC 2024 IRL
+  - GBC2024IRL
+---
+
+# GB Cup 2024 In Real Life
+
+![GB杯2024线下赛图片](img/gbc2024irl-banner.png)
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:12]:
```suggestion
![GB Cup 2024 IRL banner](img/gbc2024irl-banner.png)
```

File: wiki/Tournaments/GBC/GBC_2024_IRL/en.md [hunk 16]
```diff
@@ -0,0 +1,166 @@
...
+| Placing | Prizes |
+| :-: | :-- |
+| ![Golden Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, championship trophy, GBC pillow, GBC medal |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | KanonBot profile badge, 2nd place trophy, GBC pillow, GBC medal |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | KanonBot profile badge, 3rd place trophy, GBC pillow, GBC medal |
+| *Others* | GBC pillow, GBC medal |
+
+![Trophy image](img/gbc2024irl-cups.jpg "GBC 2024 IRL trophies")
+
+![Badge image](img/gbc2024irl-winner.png)
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_IRL/en.md:35]: mind updating the translation for this part too

```suggestion
![Badge image](img/gbc2024irl-winner.png "GBC 2024 IRL winner badge")
```

**Comment by Walavouchey** [comment]
in the future, please don't pr from the `master` branch. create a new branch before making changes. see https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits

**Comment by yobrevelc** [comment]
> in the future, please don't pr from the `master` branch. create a new branch before making changes. see https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits

ok I see

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**